### ISSUE TITLE
Gauge Properties empty when adding widget in dashboard

### ISSUE THREAD

**Issue by denniseffing** [cdjackson/HABmin2#164]
There is no configuration to choose from when adding a gauge to a dashboard:

![ss 2016-05-02 at 08 50 54](https://cloud.githubusercontent.com/assets/5879568/14948662/242094be-1043-11e6-80b3-f65cf349093a.png)

It might be the case that I am missing the correct sensors for displaying correct properties to configure.
Still, there should be an error message, stating that no correct sensor could be found.

**Comment by ifol** [comment]
Yeah, came across the same - I have one temperature item however which should be suitable for a gauge (it displays correctly in sitemap), still nothing displays in  Gauge Properties, don't know what's wrong

**Comment by miosolHH** [comment]
Same issue for me. Using org.openhab.ui.habmin_0.1.7-SNAPSHOT.jar.
